### DISCUSSION TITLE
Fish

### DISCUSSION THREAD

**Discussion by brynjsmith** [mnowotnik/fzshell#11] [Questions & Answers]
Hey, I spent a half-an-hour dinking around with trying to install this into my fish shell ~/.config or ~/.fzshell or variations on that theme, and I can't get it to work. I'm always looking for shortcuts and I write fish functions all the time, so this sounds cool in theory. I'm not getting any error messages, but I'm not getting any keybindings either. It's entirely possible that I've customized my shell to the point where this extension is unusable, but I hope not. Let's chat!

**Comment by mnowotnik** [comment]
@brynjsmith It may be a bug! You're one of early adopters. I've just created [gitter chat](https://gitter.im/fzshell/community) for fzshell, hit me up there.

**Comment by mnowotnik** [comment]
@brynjsmith There was indeed a bug if a user decided not to install automatic key bindings, but to source manually instead. It should be alright now.
